lib/rex/proto/kerberos/model/pre_auth_pac_options_flags.rb

```diff
@@ -0,0 +1,22 @@
+# -*- coding: binary -*-
+
+module Rex
+  module Proto
+    module Kerberos
+      module Model
+        # THe PA-PAC-OPTIONS KerberosFlags are represented as a bit string.
+        # This module associates the human readable name, to the index the flag value is found at within the bit string.
+        # https://www.rfc-editor.org/rfc/rfc4120.txt - KDCOptions      ::= KerberosFlags
+        module PreAuthPacOptionsFlags
```

Comment:
```suggestion
        module PreAuthPacOptionsFlags < KerberosFlags
```

Reply:
I also made it a `class` instead of a `module`.